sql/mysql/inspect.go

```diff
@@ -789,10 +789,7 @@ FROM
 WHERE
 	t2.CONSTRAINT_TYPE = 'FOREIGN KEY'
 	AND t1.TABLE_SCHEMA = ?
-	AND t1.TABLE_NAME IN (%s)
-ORDER BY
-	t1.CONSTRAINT_NAME,
-	t1.ORDINAL_POSITION`
```

Comment:
This is problematic because this ensures we get the columns in the right order. See: https://github.com/ariga/atlas/blob/master/sql/internal/sqlx/sqlx.go#L130-L135

Reply:
We can order this in the application code, but I want to make sure before this actually causes the slowness in the inspection.

Reply:
@a8m I think you figure out the root cause. The order was causing an issue with MySQL planner, I shared the explain in this ticket https://github.com/ariga/atlas/issues/1727

I found another way to resolve the issue by spliting the query and do the join in Go, but I am not a go Expert :wink: